src/utils/getResult.js

```diff
@@ -0,0 +1,35 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
+/**
+ * 컴퓨터의 숫자와 플레이어의 숫자를 비교하여 스트라이크와 볼을 얻어냅니다.
+ * @param {Object} params
+ * @param {number[]} params.computerNums - 컴퓨터의 숫자 배열
+ * @param {number[]} params.userNums - 플레이어가 입력한 숫자 배열
+ * @returns {{ strike: number, ball: number }} - 스트라이크와 볼 개수
+ */
+export async function getResult({ computerNums, userNums }) {
+  let strike = 0;
+  let ball = 0;
+  const ARR_LENGTH = Math.min(computerNums.length, userNums.length);
+
+  for (let i = 0; i < ARR_LENGTH; i++) {
+    for (let j = 0; j < ARR_LENGTH; j++) {
+      if (i === j && computerNums[i] === userNums[j]) strike++;
+      if (i !== j && computerNums[i] === userNums[j]) ball++;
+    }
+  }
```

Comment:
```suggestion
  for (let i = 0; i < ARR_LENGTH; i++) {
    for (let j = 0; j < ARR_LENGTH; j++) {
      if (i === j && computerNums[i] === userNums[j]) strike++;
      if (i !== j && computerNums[i] === userNums[j]) ball++;
    }
  }
```
혹시 이렇게 이중 반복문으로 생각하신 이유가 있으신가요??
저는 루프는 한번 돌고 같은 자리이면 `strike`, `userNums[i]`가 `computerNums` 배열에 포함되어 있으면 `ball` 카운트해주었습니다

Reply:
`includes()` 메서드를 사용하신거죠? 그렇게도 생각해봤는데 `includes`도 어쨌든 배열을 순회하는 메서드이기 때문에 이중 반복문과 큰 차이는 없을거라고 생각했습니다!
근데 `includes()`를 사용하면 가독성이 좀더 좋아질 것 같기도 하네요! 참고하겠습니다. 감사합니당 :)